tests/test_boundary_conditions.py

```diff
@@ -3,23 +3,27 @@
 
 # System imports
 import numpy as np
-from tests.test_rod.test_rods import MockTestRod
 from elastica.boundary_conditions import (
     ConstraintBase,
     FreeBC,
     OneEndFixedBC,
     FixedConstraint,
+    ConfigurableConstraint,
```

Comment:
```suggestion
    GeneralConstraint,
```

Reply:
Implemented in https://github.com/GazzolaLab/PyElastica/pull/143/commits/4c06b0e850b5d5fb647394771572c35a9f0c641e